app/services/exports/export_distributions_csv_service.rb

```diff
@@ -63,8 +65,11 @@ def base_table
         "Source Inventory" => ->(distribution) {
           distribution.storage_location.name
         },
-        "Total Items" => ->(distribution) {
-          distribution.line_items.total
+        "Total number of Items" => ->(distribution) {
+          # filter the line items by item id (for selected item filter) to
+          # get the number of items
+          distribution.line_items.find_by(item_id: @filtered_item_id)&.quantity ||
+            distribution.line_items.total
         },
         "Total Value" => ->(distribution) {
           distribution.cents_to_dollar(distribution.line_items.total_value)
```

Comment:
I know the issue didn't mention cost, but should this also include the filter logic? Might be misleading to return results that show a quantity for a subset of all items but total value of _all_ items.

Reply:
hmm, the filtered view shows that these many # items of that kind were included in the distribution, but the total value of that distribution is shown in the `Total value` column. If we only show the total value of only this item, then on the web view, they have click view on that particular distribution to find out that total value of all items. May be we could name the columns as Total value (of all items) ?